### PR TITLE
Html integration with DB and Testing

### LINKED ISSUE #5: Integrate the HTML with the DB models
- [x] Once the systems are set up, integrate them together to get the articles appearing on the website.
- [x] Update the Controller to not respond with an error message with no articles. 
- [x] Validate the page functionality still work with the DB integration
- [x] Implement unit tests for all backend code.

### PR DESCRIPTION
**Pull request by gaaliciA1990**
Updated controller to remove errors for empty DB. Ensured full functionality of HTML pages with DB code added. 
Implemented unit tests for all classes, 100% coverage, all passing.

Closes #5

### CODE REVIEW

File: src/test/kotlin/com/gamenews/ApplicationTest.kt [hunk 1]
```diff
@@ -52,4 +62,352 @@ class ApplicationTest {
...
+            response.bodyAsText().contains("Sorry, someone beat you to the punch. We couldn't find that article")
+        )
+    }
 }
+
+// SET UP
+
+// DO
+
+// ASSERT
```
Comment (CRamsan) [inline src/test/kotlin/com/gamenews/ApplicationTest.kt]: Delete

Comment (CRamsan) [reply src/test/kotlin/com/gamenews/ApplicationTest.kt]: I am surprised this was not found by detekt. I would have expected this to be caught by the tool.

**Comment by CRamsan** [comment]
Remove extra lines, everything looks great. Good job! 👍

File: src/test/kotlin/com/gamenews/ApplicationTest.kt (hunk 1 above)
Comment (gaaliciA1990) [reply src/test/kotlin/com/gamenews/ApplicationTest.kt]: Done!

**Review by CRamsan — APPROVED** [review]

**Pull request merged**